settings.gradle

```diff
@@ -11,4 +11,4 @@ plugins {
     id("org.gradle.toolchains.foojay-resolver-convention") version "0.7.0"
 }
 
-rootProject.name = "JabRef"
+rootProject.name = "jabRef"
```

Comment:
Please do not change something here, this has severe consequences for our installer infrastructure.
As the letter R is still in uppercase, I suggest this change happend by mistake.
As a general advice: Always review your own changes before committing them (use git gui or any other git tool) and only commit changes that are directly linked to your issue and are really intended.

Reply:
@Toro520 You can also check with "git blame" when this line was changed and go to the pull request for some background. This is a procedure for all changes in important part of the code!

We did that change at https://github.com/JabRef/jabref/pull/10322, but it did not work. Changing that lead to more changes, which had unforseen consquences on macOS: https://github.com/JabRef/jabref/pull/10345

Reply:
@calixtus Hi, I'm so sorry to cause this trouble, but when I first time to install it, I have this kind of issue, and I find the documentation : [Changing only the case of the Gradle root project name causes exception while importing project: "java.lang.IllegalStateException: Module entity with name: untitled20.main should be available"](https://youtrack.jetbrains.com/issue/IDEA-317606/Changing-only-the-case-of-the-Gradle-root-project-name-causes-exception-while-importing-project-java.lang.IllegalStateException) from [Pre Condition 3: Code on the local machine](https://devdocs.jabref.org/getting-into-the-code/guidelines-for-setting-up-a-local-workspace/pre-03-code.html)to solve it.  It said directory and project name should be same. Right now I'm confused about it.
But now I have fixed it. I hope everything goes well.
<img width="1080" alt="1698276554077" src="https://github.com/JabRef/jabref/assets/128112774/ef1f58a0-ba98-4b9f-a6d8-a5369d1bf725">

Reply:
@koppor Thanks so much. Because I'm not sure is that the right way to do this(pull request), so I just tried the draft one to have a practice. You are so enthusiasm and passionate about it. I must could learn a lot from you guys.